linode_api4/linode_client.py

```diff
@@ -109,45 +110,45 @@ def get_kernels(self, *filters):
         return self.client._get_and_filter(Kernel, *filters)
 
     # create things
-    def create_instance(self, ltype, region, image=None,
+    def instance_create(self, ltype, region, image=None,
             authorized_keys=None, **kwargs):
         """
-        Creates a new Linode. This function has several modes of operation:
+        Creates a new Linode Instance. This function has several modes of operation:
 
-        **Create a Linode from an Image**
+        **Create an Instance from an Image**
 
-        To create a Linode from an :any:`Image`, call `create_instance` with
+        To create an Instance from an :any:`Image`, call `instance_create` with
         a :any:`Type`, a :any:`Region`, and an :any:`Image`.  All three of
         these fields may be provided as either the ID or the appropriate object.
         In this mode, a root password will be generated and returned with the
-        new Linode object.  For example::
+        new Instance object.  For example::
 
-           new_linode, password = client.linode.create_instance(
-               "g5-standard-1",
+           new_linode, password = client.linode.instance_create(
+               "g6-standard-1",
```

Comment:
you used g6-standard-2 in other places. Do you want to change it here as well to be consistent? Doesn't really matter, but since you're in here anyway...